packages/lib/CalEventParser.ts

```diff
@@ -194,7 +194,7 @@ ${getAppsStatus(calEvent, t)}
 ${
   // TODO: Only the original attendee can make changes to the event
   // Guests cannot
-  !calEvent.seatsPerTimeSlot && getManageLink(calEvent, t)
+  calEvent.seatsPerTimeSlot ? "" : getManageLink(calEvent, t)
```

Comment:
@joeauyeung this check was printing false in the calendar event description. 

Now fixed
<img width="443" alt="Screenshot 2023-05-23 at 6 34 47 PM" src="https://github.com/calcom/cal.com/assets/53316345/1b98830d-f4c0-4759-8240-5ad195e67bb1">